src/uct/ib/mlx5/ib_mlx5_dv.c

```diff
@@ -22,6 +22,114 @@ ucs_status_t uct_ib_mlx5dv_init_obj(uct_ib_mlx5dv_t *obj, uint64_t type)
 }
 #endif
 
+#if HAVE_MLX5_DEVICE
+UCT_DEVICE_SPEC(0x02c9, 4113, "Connect-IB",
```

Comment:
is there a way to unite this with ib_device.c to avoid code duplication?

Reply:
Require new level of abstraction, let's ask for another opinion.